packages/roosterjs-editor-plugins/lib/plugins/AutoFormat/AutoFormat.ts

```diff
@@ -56,47 +63,84 @@ export default class AutoFormat implements EditorPlugin {
                 this.lastKeyTyped = '';
             }
 
+            const searcher = this.editor.getContentSearcherOfCursor(event);
```

Comment:
It seems this will run for all key press, no matter what key is. I'm a little concerned about the perf.